e2e.bats

```diff
@@ -0,0 +1,53 @@
+#!/usr/bin/env bats
+
+@test "Accept service account with no risky permissions" {
+	run kwctl run  --request-path test_data/pod_creation.json  --allow-context-aware \
+		--settings-path test_data/settings.json \
+		--replay-host-capabilities-interactions test_data/can_i_returns_not_allowed.yml annotated-policy.wasm
```

Comment:
shouldn't this be consuming `test_data/can_i_returns_allowed.yml`?

Reply:
No, that's right. the `test_data/can_i_returns_allowed` means: "The Kubernetes API returned that the service account is allowed to perform the operation from the given request". As the policy wants to block allowed operations, this is the right thing. In other words, if the Kubernetes says "it's allowed", the policy rejects. If Kubernetes says "not allowed", the policy accepts.